Script/AtomicEditor/ui/Shortcuts.ts

```diff
@@ -99,10 +99,13 @@ class Shortcuts extends Atomic.ScriptObject {
 
     }
 
-    invokeGizmoAxisModeChanged(mode: Editor.AxisMode, toggle: boolean = false) {
-
-        this.sendEvent("GizmoAxisModeChanged", { mode: mode, toggle: toggle });
+    toggleGizmoAxisMode() {
+        var editor = EditorUI.getMainFrame().resourceframe.currentResourceEditor;
```

Comment:
I wonder if we shouldn't have a more direct way of accessing the current resource editor?  Could be a follow up